registry/base_configuration_listener.go

```diff
@@ -82,14 +82,14 @@ func (bcl *BaseConfigurationListener) genConfiguratorFromRawRule(rawConfig strin
 	return nil
 }
 
-// OverrideUrl ...
+// OverrideUrl gets existing configuration rule and override provider url before exporting.
```

Comment:
OverrideUrl gets existing configuration rule and overrides provider url before exporting.